blockstore/apps/api/v1/tests/test_contract.py

```diff
@@ -233,6 +236,70 @@ def test_list(self):
         assert len(list_data) == 1
 
 
+class BundleVersionsTestCase(ApiTestCase):
+    """Test basic BundleVersions functionality."""
+    def setUp(self):
+        super().setUp()
+        collection_response = self.client.post(
+            '/api/v1/collections',
+            data={'title': 'Bundle Default Collection 😀'}
+        )
+        collection_data = response_data(collection_response)
+
+        bundle_response = self.client.post(
+            '/api/v1/bundles',
+            data={
+                'collection_uuid': collection_data['uuid'],
+                'description': "Draft Test 😀😀😀😀 Bundle",
+                'slug': 'draft_test',
+                'title': "Draft Test Bundle 😀"
+            }
+        )
+        self.bundle_data = response_data(bundle_response)
+
+    def test_redirect(self):
+        create_draft_response = self.client.post(
+            '/api/v1/drafts',
+            {
+                'bundle_uuid': self.bundle_data['uuid'],
+                'name': 'studio_draft',
+                'title': "For DraftsTest.test_basic_draft_commit",
+            }
+        )
+        draft_data = response_data(create_draft_response)
+        self.client.patch(
+            draft_data['url'],
+            data={
+                'files': {
+                    'hello.txt': encode_str_for_draft("Hello World! 😀")
+                }
+            },
+            format='json'
+        )
+
+        self.client.post(u'/api/v1/drafts/{}/commit'.format(draft_data["uuid"]))
+        url = reverse(
+            "api:v1:bundleversion-detail",
+            kwargs={"bundle_uuid": draft_data['bundle_uuid'], 'version_num': 1},
+        )
+        detail_response = self.client.get(url)
+        detail_data = response_data(detail_response)
+
+        response_redirect_url = detail_data['snapshot']['files']['hello.txt']['url']
+        parsed_url = urlparse(response_redirect_url)._replace(netloc='', scheme='')
+
+        url = reverse(
+            "api:v1:bundleversion-redirect",
```

Comment:
URLs should be hard-coded into tests, not reversed, to ensure the tests will break if we change URLs (breaking API backwards compatibility)

Reply:
makes sense, improved